src/main/java/org/openrewrite/staticanalysis/MethodNameCasing.java

```diff
@@ -113,37 +104,9 @@ public J.MethodDeclaration visitMethodDeclaration(J.MethodDeclaration method, Ex
                     !method.isConstructor() &&
                     !simpleName.startsWith("_") &&
                     !STANDARD_METHOD_NAME.matcher(simpleName).matches()) {
```

Comment:
Keeping this matching here seems wrong to me. That should somehow also be part of the `NamingService`.